api/storage_service.cc

```diff
@@ -541,6 +541,25 @@ void set_storage_service(http_context& ctx, routes& r) {
         });
     });
 
+    ss::trigger_offstrategy_compaction.set(r, [&ctx](std::unique_ptr<request> req) {
+        auto keyspace = validate_keyspace(ctx, req->param);
+        auto table_names = split_cf(req->get_query_param("table"));
+        if (table_names.empty()) {
+            table_names = map_keys(ctx.db.local().find_keyspace(keyspace).metadata().get()->cf_meta_data());
+        }
+
+        return ctx.db.invoke_on_all([keyspace = std::move(keyspace), table_names = std::move(table_names)] (database& db) {
+            for (auto tname : table_names) {
+                auto& t = db.find_column_family(keyspace, tname);
+                if (t.is_pending_offstrategy_compaction()) {
```

Comment:
It seems to me is_pending_offstrategy_compaction() isn't buying us anything because repair can still be in progress, so offstrategy can be triggered prematurely. 

not sure I like this API as it can be easily misused. efficiency problems will happen if user runs it prematurely, before repair completed. @asias could we add a repair cmd instead which would be sent on completion by user. so offstrategy is triggered, if either the cmd was received, or the timer is triggered, whichever happens first.